test/report_engine_Test.php

```diff
@@ -30,9 +30,15 @@ public function report_test_files_provider() {
 				$db_pass = $connection['pass'];
 			if (isset($connection['host']) && $connection['host'])
 				$db_host = $connection['host'];
-			if (isset($connection['type']) && $connection['type'])
+			if (isset($connection['type']) && $connection['type']) {
 				$db_type = $connection['type'];
+                // merlin import doesn't understand mysqli
```

Comment:
Indentation

Reply:
Also L36-L39. Check the "files changed" in Github, to make it clear what lines are off.

The file is using tabs rather than spaces, might look OK in your editor I depending on how tabs vs spaces are rendered.